exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function here
 
+function createLongGreeting(name, number) {
+  return "Hello, my name is" + name + "and I'm " + number + " years old.";
```

Comment:
This looks good, but you might want to check if you're leaving a space before and after the name. Also, this might be a good place to try out _string interpolation_. Have a look back at the slides or read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals